cetmix_tower_server/models/cx_tower_plan_log.py

```diff
@@ -64,7 +64,13 @@ class CxTowerPlanLog(models.Model):
         "-304 if plan line reference is missing",
     )
     parent_flight_plan_log_id = fields.Many2one(
-        "cx.tower.plan.log", string="Main Log", ondelete="cascade"
+        "cx.tower.plan.log",
+        string="Main Log",
+        related="parent_command_log_id.plan_log_id",
+    )
+    parent_command_log_id = fields.Many2one(
+        "cx.tower.command.log",
+        readonly=True,
```

Comment:
Just for the better readability:

`help="Command that triggered this flight plan"`

Also `ondelete="cascade"`

Reply:
fixed